src/module/Ui/templates/static/de/datenschutz/revision-20181201.twig

```diff
@@ -0,0 +1,483 @@
+            überprüft nach dem Einloggen automatisch Deinen Kontostand und führt die Überweisung an uns mit Hilfe der
+            von Dir übermittelten TAN durch. Anschließend übermittelt sie uns unverzüglich eine Transaktionsbestätigung.
+            Nach dem Einloggen werden außerdem Deine Umsätze, der Kreditrahmen des Dispokredits und das Vorhandensein
+            anderer Konten sowie deren Bestände automatisiert geprüft.</p>
+        <p>Neben der PIN und der TAN werden auch die von Dir eingegebenen Zahlungsdaten sowie Daten zu Deiner Person an
+            die Sofort GmbH übermittelt. Bei den Daten zu Deiner Person handelt es sich um Vor- und Nachname, Adresse,
+            Telefonnummer(n), Email-Adresse, IP-Adresse und ggf. weitere zur Zahlungsabwicklung erforderliche Daten. Die
+            Übermittlung dieser Daten ist notwendig, um Deine Identität zweifelsfrei zu festzustellen und
+            Betrugsversuchen vorzubeugen.</p>
+        <p>Die Übermittlung Deiner Daten an die Sofort GmbH erfolgt auf Grundlage von Art. 6 Abs. 1 lit. a DSGVO
+            (Einwilligung) und Art. 6 Abs. 1 lit. b DSGVO (Verarbeitung zur Erfüllung eines Vertrags). Du hast die
+            Möglichkeit, Deine Einwilligung zur Datenverarbeitung jederzeit zu widerrufen. Ein Widerruf wirkt sich auf
+            die Wirksamkeit von in der Vergangenheit liegenden Datenverarbeitungsvorgängen nicht aus.</p>
+        <p>Details zur Zahlung mit Sofortüberweisung entnehmen Sie folgenden Links: <a
+                href="https://www.sofort.de/datenschutz.html">https://www.sofort.de/datenschutz.html</a> und <a
+                href="https://www.klarna.com/sofort/">https://www.klarna.com/sofort/</a>.</p>
+        <h4>Stripe inc.</h4>
+        <p>Diese Webseite nutzt zur Abwicklung der Spende Dienstleistungen des Technologieunternehmens Stripe inc.
+            („Stripe”), 510 Townsend Street, San Francisco, CA 94103. Die bei Ausfüllen des Spendenauftrags angegebenen
+            Daten, d.h. Höhe und Rhythmus der Spende sowie Deine Kontaktdaten werden auf Servern von Stripe in den USA
+            gespeichert. Stripe nutzt diese Daten zur Durchführung der Transaktion, sowie zur Durchführung von Maßnahmen
+            gegen Betrugsabwehr. </p>
+        <p>Die Übermittlung Deiner Daten an Stripe erfolgt auf Grundlage von Art. 6 Abs. 1 lit. a DSGVO (Einwilligung)
+            und Art. 6 Abs. 1 lit. b DSGVO (Verarbeitung zur Erfüllung eines Vertrags). Du hast die Möglichkeit, Deine
+            Einwilligung zur Datenverarbeitung jederzeit zu widerrufen. Ein Widerruf wirkt sich auf die Wirksamkeit von
+            in der Vergangenheit liegenden Datenverarbeitungsvorgängen nicht aus.</p>
+        <p>Die Datenschutzerklärung von Stripe findest Du <a href="https://stripe.com/de/privacy">hier</a>.</p>
+        <h4>twingle</h4>
+        <p>Diese Webseite nutzt zur Verarbeitung von Spendenaufträgen twingle, eine Online Fundraising Software
```

Comment:
Did you talk with Miriam about these changes? Or better: did someone check, that the collected data is correct regarding twingle?